### PR TITLE
Python : Attribute Error in Competitive programming... #8116

### LINKED ISSUE #8116: Python :  Attribute Error in Competitive programming...
### Description

What is AttributeError and How to overcome from this error in Python.

### Domain

Competitive Programming

### Type of Contribution

Documentation

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.

### PR DESCRIPTION
**Pull request by anjalikumawat2002**
<hr>

## Description 📜

What is AttributeError and How to overcome from this error in Python.
Fixes #8116 

<hr>

## Type of change 📝

- [ ] Au#dio (Should be in mp3 format Includes speech clarity, Concise ,Low distortion)
- [ ] Vi#deo (Animations, screen-recordings, presentations and regular explanatory films are all possibilties etc)
- [x] Documentation (Content Creation in the form of codes or tutorials)
- [ ] Other (If you choose other, Please mention changes below) 

<hr>

## Domain of Contribution 📊

- [ ] Android Dev #(Flutter)
- [ ] Android Dev #(Java)
- [ ] Android Dev #(Kotlin)
- [ ] Backend Dev #(Java)
- [ ] Backend Dev #(.NET)
- [ ] Backend Dev #(PHP)
- [ ] Backend Dev #(Python)
- [ ] C/#CPP
- [x] Competitive Programming
- [ ] Cyber #Security
- [ ] DS#A
- [ ] Data#base
- [ ] Datascience with #Python
- [ ] Datascience with #R
- [ ] Frontend Dev #HTML/CSS/JS
- [ ] Frontend Dev #React/Angular/Vue
- [ ] Go#lang
- [ ] Interview #Prep
- [ ] Java_#Domain
- [ ] JavaScript_#Domain
- [ ] ME#RN
- [ ] Machine #Learning
- [ ] Open #Source
- [ ] Python_#Domain
- [ ] Ru#st
- [ ] Statis#tics
- [ ] UI/#UX

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

**Review by ravireddy07 — COMMENTED** [review]
@anjalikumawat2002 Actually, you can prepare a doc and cover all type of Possible Error/Exception that anyone can face during Competitive Programming.
This Doc seems like too small and around very less information.

**Comment by anjalikumawat2002** [comment]
> @anjalikumawat2002 Actually, you can prepare a doc and cover all type of Possible Error/Exception that anyone can face during Competitive Programming. This Doc seems like too small and around very less information.

okk sir

**Comment by anjalikumawat2002** [comment]
> > @anjalikumawat2002 Actually, you can prepare a doc and cover all type of Possible Error/Exception that anyone can face during Competitive Programming. This Doc seems like too small and around very less information.
> 
> okk sir
> 

@ravireddy07 sir please review my PR..

File: Competitive_Programming/Python/AttributeError/AttributeError.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+AttributeError: Integers cannot 'append'.
+```
+
+To see a list of all function and data attributes associated with an object of a given type, simply call the built-in function `help()` on an object of that type in an interactive Python session.
+
+Or, if you want to see a fairly comprehensive overview of types and their attributes, check out the Python documentation on [built-in types](http://docs.python.org/2/library/stdtypes.html).
+
+
+## **Conclusion**
```
Comment (ravireddy07) [inline Competitive_Programming/Python/AttributeError/AttributeError.md:109]: By default headers will be in Bold, so here this asterisk's won't work.

**Review by ravireddy07 — CHANGES_REQUESTED** [review]
change the file/doc name

File: Competitive_Programming/Python/AttributeError/AttributeError.md (hunk 1 above)
Comment (ErzaTitania-2001) [reply Competitive_Programming/Python/AttributeError/AttributeError.md:109]: @ravireddy07 once it is approved, do relay to me the information. :)

**Comment by anjalikumawat2002** [comment]
> change the file/doc name

@ravireddy07 sir Please suggest me that what should I name the file??

**Comment by anjalikumawat2002** [comment]
> change the file/doc name

Done @ravireddy07 sir please check..

File: Competitive_Programming/Python/AttributeError/AttributeError.md (hunk 1 above)
Comment (ravireddy07) [reply Competitive_Programming/Python/AttributeError/AttributeError.md]: sure @ErzaTitania-2001

**Comment by ravireddy07** [comment]
> > change the file/doc name
> 
> Done @ravireddy07 sir please check..

You would have changed folder name also...The folder name is still AttributeError

**Comment by anjalikumawat2002** [comment]
> > > change the file/doc name
> > 
> > 
> > Done @ravireddy07 sir please check..
> 
> You would have changed folder name also...The folder name is still AttributeError @anjalikumawat2002

Done @ravireddy07 sir please check ..

**Review by ravireddy07 — APPROVED** [review]

File: Competitive_Programming/Python/AttributeError/AttributeError.md (hunk 1 above)
Comment (ravireddy07) [reply Competitive_Programming/Python/AttributeError/AttributeError.md]: You can merge now @ErzaTitania-2001  :)

Comment (ErzaTitania-2001) [reply Competitive_Programming/Python/AttributeError/AttributeError.md]: Sure

**Pull request merged**